src/main/java/com/revature/services/EvalLogicImpl.java

```diff
@@ -16,40 +16,76 @@
 
 	@Autowired
 	private EvalRepository dao;
+	
+
+//CREATE----------------------------
 
+//RETRIEVE--------------------------
 	@Override
 	public Eval getEvalById(int id) {
 		Eval eval = dao.findOne(id);
-		System.out.println(eval);
+		try {
+			System.out.println(eval);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)